### DISCUSSION TITLE
How to apply my own patches

### DISCUSSION THREAD

**Discussion by mvmx3** [ayamir/nvimdots#1177] [General]
我有一些patch,每次更新的时候需要打上。如果这个github库有更新的话，就打不上来，需要稍微调整一下才能打上。
是否可以有一个配置文件，去覆盖这个github 库的功能，就像vscode 的用户配置一样。这样就不需要更新之后把自己的patch 打上。

**Comment by charliie-dev** [comment]
https://github.com/ayamir/nvimdots/wiki/Usage#how-to-customize-according-to-your-preferences
